cms/templates/settings.html

```diff
@@ -94,10 +96,10 @@ <h3 class="title">${_("Course Summary Page")} <span class="tip">${_("(for studen
               <li class="action-item">
                 <%
                   email_subject = urllib.quote(_("Enroll in {course_display_name}").format(
-                      course_display_name = context_course.display_name_with_default_escaped
+                      course_display_name = context_course.display_name_with_default
```

Comment:
There are some other issues in this block that @robrap pointed out [here](https://github.com/edx/edx-platform/pull/12042/files#r62843734). I decided not to tackle those issues and leave them for #12042, as I don't think they will impact the Bok Choy XSS tests.